core/src/main/web/app/mainapp/services/notebookcellmodelmanager.js

```diff
@@ -443,8 +443,9 @@
               if(!runningJob || runningJob.cellId !== cell.id){
                 cell.output.result = undefined;
                 cell.output.elapsedTime = undefined;
+                cell.output.height = undefined;
```

Comment:
why are we initializing to undefined? It's an object property, it will be undefined by default.